js/ui/file_manager/ui.file_manager.commands.js

```diff
@@ -0,0 +1,90 @@
+let tryCreateFolder = function(fileManager) {
```

Comment:
if tryCreateFolder isn't changing then use `const` instead of `let`

Reply:
We plan to refactor this code during the implementation of the next feature - toolbar support